toolchain/driver/clang_runner_test.cpp

```diff
@@ -181,5 +181,38 @@ TEST(ClangRunnerTest, DashC) {
   EXPECT_THAT(err, StrEq(""));
 }
 
+TEST(ClangRunnerTest, BuitinHeaders) {
+  std::filesystem::path test_file = WriteTestFile("test.c", R"cpp(
+#include <stdalign.h>
```

Comment:
In clang_configuration.bzl, we're detecting std with:

```
"""
#if __has_include(<version>)
#include <version>
#endif
#ifndef _LIBCPP_STD_VER
#error "No libc++ install found!"
#endif
"""
```

Is this something that we should be trying to share here? Or if not, why made you prefer `stdalign.h` here versus `version`?

Reply:
So, `version` detects libc++ being installed.

Here, we're trying to detect that Clang's compiler-builtin headers are installed. `stdalign.h`, perhaps weirdly, is a compiler-builtin header, and not part of libc++ or even the underlying system's C standard library.